app/models/miq_server/worker_management/kubernetes.rb

```diff
@@ -17,7 +40,7 @@ def cleanup_failed_deployments
 
   def failed_deployments(restart_count = 5)
     # TODO: This logic might flag deployments that are hitting memory/cpu limits or otherwise not really 'failed'
-    current_pods.values.select { |h| h[:last_state_terminated] && h.fetch(:container_restarts, 0) > restart_count }.collect { |h| h[:label_name] }
+    current_pods.values.select { |h| h[:last_state_terminated] && h.fetch(:container_restarts, 0) > restart_count }.pluck(:label_name)
```

Comment:
When did they add .pluck to Array? 😮

Reply:
A while? https://github.com/rails/rails/pull/20350 :)

Reply:
How have I never known this? 🤯